.github/workflows/ci-e2e-cassandra.yml

```diff
@@ -22,14 +22,19 @@ jobs:
       fail-fast: false
       matrix:
         jaeger-version: [v1, v2]
+        skip-apply-schema: [true, false]
         version:
         - distribution: cassandra
           major: 4.x
           schema: v004
         - distribution: cassandra
           major: 5.x
           schema: v004
-    name: ${{ matrix.version.distribution }} ${{ matrix.version.major }} ${{ matrix.jaeger-version }}
+        exclude:
+        # Exclude v1 as create schema on fly is available for v2 only
+        - jaeger-version: v1
+          skip-apply-schema: true
+    name: ${{ matrix.version.distribution }} ${{ matrix.version.major }} ${{ matrix.jaeger-version }} ${{ matrix.skip-apply-schema == true && 'auto' || 'manual' }}
```

Comment:
`skip-apply-schema` is the inverse meaning of CASSANDRA_CREATE_SCHEMA. Why can't we do it the other way: have a matrix param `schema: [manual, auto]` and translate that into correct true/false value for CASSANDRA_CREATE_SCHEMA?

Reply:
> skip-apply-schema is the inverse meaning of CASSANDRA_CREATE_SCHEMA.

Not an inverse actually. `skip-apply-schema` means skip schema creation on running integration scripts. and `CASSANDRA_CREATE_SCHEMA` means create schema using code.
So, basically if `skip-apply-schema == true`, `CASSANDRA_CREATE_SHEMA` would also have to be true, so that tables are created for running tests.